### ISSUE TITLE
Options for print dissapered from menues

### ISSUE THREAD

**Issue by KpokPatagon** [MrTarantula/WIPrint#44]
After activating the new Azure DevOps backlog experience, the print option is not shown in menues, so there is no way to print anything.

**Comment by mradckeIRT** [comment]
To clarify, this occurs in the new web platform Boards hubs.

**Comment by KpokPatagon** [comment]
Yes, that is correct. The print option dissapered, from queries too, as shown in the image pasted below.
But the configuration option in the main menú is still there.

![image](https://user-images.githubusercontent.com/4146638/196144180-9a5ea942-ad14-4fdc-b565-9a4eb7595f3b.png)


***@***.***

De: ***@***.***>
Enviado: lunes, 17 de octubre de 2022 05:26
Para: ***@***.***>
CC: Marcelo ***@***.***>; ***@***.***>
Asunto: Re: [MrTarantula/WIPrint] Options for print dissapered from menues (Issue #44)


To clarify, this occurs in the new web platform Boards hubs.

**Comment by KpokPatagon** [comment]
Hello, 
I found this in the console.

`Ignored invalid external menu item: mrtarantula.wiprint.print-work-item`

In case it helps.

**Comment by Manuel-Cabrera** [comment]
Hello. I have the same issue, just after migrating our collection from Server to Services. Hope it will be solved soon, we used it a lot.

**Comment by huastekotester** [comment]
Please make the print option to show up within the New Board Hubs experience. As of 2023-11-07, this new experience is in preview and we can de-activate it but MS may make it the standard soon.

**Comment by VictorioBerra** [comment]
Hello same issue. @MrTarantula  !!!

**Comment by BKanters** [comment]
Hi,

We are currently also experiencing this issue, while not having problems before. At the moment no printing is possible because the option does not become available in the menu.

Hopefully it can be fixed, is there a workaround that somehow we are still able to get prints?

**Comment by deepaly123** [comment]
Turn Off the preview (new board hub toggle under settings) and the items reappears..

**Comment by RoyalWulf** [comment]
Is this extension still being maintained? the New Boards hub feature is being rolled out ad we wont have the option to turn off

**Comment by GaryHendrickson** [comment]
We're experiencing the same issue and we have several people who depend on this functionality.

**Comment by surensivalingam** [comment]
We are having the same issue, does anyone have a fix for this issue?

**Comment by surensivalingam** [comment]
> We're experiencing the same issue and we have several people who depend on this functionality.

Any luck?

**Comment by Manuel-Cabrera** [comment]
Hello MrTarantula, anyone is maintaining this extension? Do we have to look for another option to print work items?

**Comment by manu-cabrera** [comment]
Hello all. After some research, we found the reason the menues are not beign rendered in the menues in Azure Devops Services. The problem occurs when New Boards feature is enabled for the user / organization.

Going to Preview Features:
![image](https://github.com/MrTarantula/WIPrint/assets/163137520/753fd4e6-6e2d-4989-99e3-cc4915e92923)

and disabling New Boards Hubs setting:
![image](https://github.com/MrTarantula/WIPrint/assets/163137520/1cd1aaca-9c61-4d01-b74c-79c4e35c201a)

worked for me. Hope this helps!

Anyone is able to fix it in the repo? Other way to do it is fork and use the fix in our org but I guess it should be better to fix it and make it available to everyone.

Regards,
Manuel
